TINY-9193: The caret is moved to the previous line after Enter key pressed (#8234)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- Dragging a noneditable element before or after another noneditable element now works correctly. #TINY-9253
- The restored selection after a redo or undo action was not scrolled into view. #TINY-9222
- A newline could not be inserted when the selection was restored from a bookmark after an inline `contenteditable="false"` element. #TINY-9194

> - The caret was moved to the previous line when the inline text pattern have been executed on Enter key pressing in Firefox. #TINY-9193

Reviewer: Not sure if this is better:

```suggestion
- The caret was moved to the previous line when a text pattern executed a `mceInsertContent` command on Enter key when running on Firefox. #TINY-9193
```
